src/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>React FlashCards</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.css" />
+  <link href="https://fonts.googleapis.com/css?family=Caveat|Kalam" rel="stylesheet"/>
```

Comment:
If you really want a more custom feel to your UI, you can choose a theme from here: https://bootswatch.com/
And link it from here: https://cdnjs.com/libraries/bootswatch

Avoid custom styles for this project. We want to concentrate on functionality and, unfortunately, you'll be judged on how it looks even though you haven't studied design.